src/aat/java/uk/gov/hmcts/ccd/userprofile/befta/UserProfileTestAutomationAdapter.java

```diff
@@ -1,16 +1,7 @@
 package uk.gov.hmcts.ccd.userprofile.befta;
 
 import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
-import uk.gov.hmcts.befta.dse.ccd.TestDataLoaderToDefinitionStore;
 
 public class UserProfileTestAutomationAdapter extends DefaultTestAutomationAdapter {
```

Comment:
it looks like we missed to override the below method:

    **@Override
    protected BeftaTestDataLoader buildTestDataLoader() {
      ... .... 
    }**
    
    @rajaram-nagarajan  - Could you please raise a quick PR for this fix?

Reply:
@rajaram-nagarajan ^^^